src/components/BookCard.jsx

```diff
@@ -6,14 +6,14 @@ import ProgressBar from './ProgressBar';
 const BookCard = ({ book }) => (
   <div className="book__card">
     <div className="book__info">
-      <span className="book__categories">{ book.categorie }</span>
-      <h3 className="book__title">{book.name}</h3>
+      <span className="book__categories">{ book.category }</span>
+      <h3 className="book__title">{book.title}</h3>
       <p className="book__author">{book.author}</p>
 
       <ul className="book__edit">
-        <li><ManipulateBook id={book.id} title="Comments" /></li>
-        <li><ManipulateBook id={book.id} title="Remove" /></li>
-        <li><ManipulateBook id={book.id} title="Edit" /></li>
+        <li><ManipulateBook id={book.item_id} title="Comments" /></li>
+        <li><ManipulateBook id={book.item_id} title="Remove" /></li>
+        <li><ManipulateBook id={book.item_id} title="Edit" /></li>
       </ul>
     </div>
     <div className="book__progress">
```

Comment:
- [ ] We are required to make an exact copy of the design, including fonts and buttons and spacing. Kindly replicate the design as we have it. Kindly use the correct font styles and sizes for the buttons and text.

- [ ] Kindly use the correct font weight where appropriate

- [ ] Kindly place the inputs on the same line as we have in the design.

- [ ] Kindly reduce the button border radius as we have in the design.

  your app design | zeplin design
  :---:|:---:
  ![dud](https://user-images.githubusercontent.com/54848363/208078453-578aa742-c9e4-4758-8266-e8ce7303f564.jpg) | ![dhdud](https://user-images.githubusercontent.com/54848363/208078446-ad8a57ce-2264-4362-8692-640c44bc0605.jpg)

Reply:
@DammyShittu thanks for the comment, Can you tell me where is Author in the design, because I see only title and dropdown, thanks

Reply:
Hi @wassimchakib from your design you have an author for your form, close to the title